test/common/README.md

```diff
@@ -120,11 +120,28 @@ an expected warning does not have a code then `common.noWarnCode` can be used
 to indicate this.
 
 ### fileExists(pathname)
-* pathname [&lt;string>]
+* `pathname` [&lt;string>]
 * return [&lt;boolean>]
 
 Checks if `pathname` exists
```

Comment:
Should here be a period? (and in some other places as well.)

Reply:
If so, that would be a large diff that I would prefer to save for another PR. Personally, I'd prefer no period at the end of ines like this, but as always, consistency would trump my preference. :-D